scripts/check_macos.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Checks that  README.md and the macos github action are up to date with macos_instructions.yml
+
+# This script must be run in the root of the directory where you cloned the dockstore/dockstore repo
+
+CURRENT_FILE_NAME_README=README.md
+TEMP_FILE_NAME_README=README-TEMP.md
+CURRENT_FILE_NAME_ACTION=.github/workflows/macos_installation_instructions.yml
+TEMP_FILE_NAME_ACTION=macos_installation_instructions-TEMP.yml
+
+
+cp "$CURRENT_FILE_NAME_README" "$TEMP_FILE_NAME_README"
+mv "$CURRENT_FILE_NAME_ACTION" "$TEMP_FILE_NAME_ACTION"
+
+./scripts/macos-instructions.sh
+EXIT_CODE=0
+cmp -s "$CURRENT_FILE_NAME_README" "$TEMP_FILE_NAME_README"
+if [ $? -ne 0 ]; then
+  echo "Your README is not up to date with macos_instructions.yml"
+  echo "Please generate any instructions related to Mac installation with scripts/macos-instructions.sh"
+  EXIT_CODE=1
+fi
+
+cmp -s "$CURRENT_FILE_NAME_ACTION" "$TEMP_FILE_NAME_ACTION"
```

Comment:
Hmm, with the renaming on line 14, this compares the temp file with itself so it's always identical

Reply:
Reply to **scripts/check_macos.sh** (It's not letting me comment on that thread),

I have corrected this issue.  [96e8f44](https://github.com/dockstore/dockstore/pull/5043/commits/96e8f44ee08df6de47be9031f5916e32cf8e26b1)

Reply to this conversation.

The script `./scripts/macos-instructions.sh` modifies `README.md` directly, so line 14 is creating a copy of `README.md` and then applying `./scripts/macos-instructions.sh` to `README.md` and then comparing `README.md` to the copy to ensure they are identical.